actor/builtin/miner/miner_test.go

```diff
@@ -1063,6 +1074,48 @@ func TestLatePoStFee(t *testing.T) {
 	})
 }
 
+func TestMinerGetPoStState(t *testing.T) {
+	tf.UnitTest(t)
+
+	firstCommitBlockHeight := uint64(3)
+
+	t.Run("is reported as not late within the proving period", func(t *testing.T) {
+		mal := setupMinerActorLiason(t)
+		mal.requireCommit(firstCommitBlockHeight, uint64(1))
+		mal.requireCommit(firstCommitBlockHeight+1, uint64(2))
+		mal.requireCommit(firstCommitBlockHeight+2, uint64(17))
+
+		// submit PoSt to update proving set with no done sectors
+		done := types.EmptyIntSet()
+		mal.requirePoSt(firstCommitBlockHeight+5, done)
+		mal.assertPoStStateAtHeight(PoStStateWithinProvingPeriod, firstCommitBlockHeight)
+		mal.assertPoStStateAtHeight(PoStStateWithinProvingPeriod, firstCommitBlockHeight+1)
+		mal.assertPoStStateAtHeight(PoStStateWithinProvingPeriod, firstCommitBlockHeight+6)
+	})
+
+	t.Run("is reported as PoStStateAfterProvingPeriod after the proving period", func(t *testing.T) {
+		mal := setupMinerActorLiason(t)
+		mal.requireCommit(firstCommitBlockHeight, uint64(1))
+
+		mal.assertPoStStateAtHeight(PoStStateAfterProvingPeriod, firstCommitBlockHeight+LargestSectorSizeProvingPeriodBlocks+1)
```

Comment:
something I found helpful for readability from other tests in this package is defining `firstCommitBlockHeight+LargestSectorSizeProvingPeriodBlocks` as a constant `lastHeightOfFirstPeriod` or something next to `firstCommitBlockHeight`.

Reply:
on line 1106 you could use `lastHeightOfSecondPeriod` etc